app/models/institution_builder.rb

```diff
@@ -438,4 +439,36 @@ def self.add_school_closure(version_number)
 
     Institution.connection.update(str)
   end
+
+  def self.build_zip_code_rates_from_weams(version_number)
```

Comment:
Can we get a spec for this in `institution_builder_spec.rb`?

Reply:
@jholton - spec'd!